lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -390,6 +394,38 @@ func (handler *ContainersHandlersImpl) ContainerWaitHandler(params containers.Co
 	}
 }
 
+func (handler *ContainersHandlersImpl) RenameContainerHandler(params containers.ContainerRenameParams) middleware.Responder {
+	defer trace.End(trace.Begin(fmt.Sprintf("Rename container to %s", params.Name)))
```

Comment:
At this point the container's old name is not passed into the renamecontainerhandler as a parameter